### PR TITLE
UniArts reserve SS58 address id 38

### PR DESCRIPTION
**Pull request by tuminfei**
Hello,

This PR adds a new ss58 address 38 for UniArts network.

Thank you!

### CODE REVIEW

**Review by bkchr — APPROVED** [review]

File: ss58-registry.json
Comment (bkchr) [inline ss58-registry.json:306]:
```suggestion
			"decimals": [12, 12],
```

I assume both have 12 decimals?

**Comment by tuminfei** [comment]
thx @bkchr

**Comment by bkchr** [comment]
Please merge master to fix complication issues

**Comment by bkchr** [comment]
You did not merge master properly. Please fix this.

**Comment by bigxiang** [comment]
@bkchr I rebased the pr against `substrate/master`, please see if it's okay this time, thank you!

**Comment by bkchr** [comment]
Yes ty

File: ss58-registry.json
Comment (joepetrowski) [inline ss58-registry.json:305]:
```suggestion
			"symbols": ["UART", "UINK"],
```

**Review by joepetrowski — APPROVED** [review]

**Comment by gnunicorn** [comment]
This has the needed  approves but needs a merge with master before it can be merged!

**Comment by bigxiang** [comment]
@gnunicorn While waiting for being merged, 38 was taken by another chain, so we changed to 45, please have a look and see if you have any questions, thank you!

File: ss58-registry.json
Comment (joepetrowski) [inline ss58-registry.json:376]:
```suggestion
			"symbols": ["UART", "UINK"],
```

**Review by joepetrowski — APPROVED** [review]

**Comment by gnunicorn** [comment]
@bigxiang great. please apply the changes from joe and we can merge (unfortunately, you didn't allow us to edit your branch when sending the PR or we'd have done that directly :) )

**Comment by bigxiang** [comment]
Hi @gnunicorn @joepetrowski , change applied, please have another look, thank you!

**Comment by gnunicorn** [comment]
bot merge

File: ss58-registry.json
Comment (jacogr) [inline ss58-registry.json:378]: This is missing the "network" key, see https://github.com/uni-arts-chain/substrate/blob/9723141d4e5b6c6b5006809fdfaf76a189b9588f/ss58-registry.json#L5

Comment (joepetrowski) [reply ss58-registry.json:378]: Damnit, that's my fault for missing in review. I assume it's `uniarts`, @tuminfei - can you confirm and make a followup PR?

Comment (bigxiang) [reply ss58-registry.json:378]: Oh @jacogr @joepetrowski , sorry I missed that, I will add it.

**Pull request merged**
